chanzuckerberg/single-cell-data-portal
# feat(cell-cards): Add Marker Score Feature to CellGuide

## Reason for Change

- #5098

## Changes

- add
- remove
- modify
    - Styling for Canonical/Computational tabs for Marker Genes table

## Testing steps

1. https://localhost:3000/cell-guide/CL_0000540
2. Scroll to Marker Genes section
3. Click between Computational and Canonical and ensure that the active tab is highlighted blue

## Notes for Reviewer
